policykit/templates/no-code/main.html

```diff
@@ -237,6 +237,10 @@ <h5 class="modal-title prompt small">Discard the Policy</h5>
 {% block step_scripts %}
 
 <script>
+
+    const policytemplate = JSON.parse('{{policytemplate | safe}}');
+    console.log('policytemplate', policytemplate)
```

Comment:
do you want to keep this log statement in?

Reply:
yeah, I kind of want to because it represents the middle product and can be very useful for debugging